Allow population of either an XML or docx genome note template (#110)
sanger-tol/genomenote: docs/usage.md

   --genome_notes_api '[URL for Genome Notes Portal API]'
```

If you want to populate a genome notes template file with the key-value pairs generated by this pipeline you will need to pass the path to the template file as the "note_template" parameter. Templates may be either docx or xml format.

```bash

>    --note_template '[URL for Genome Notes Portal API]'

Reviewer: ```suggestion
   --note_template '[Path to the template file]'
```
